main.cpp

```diff
@@ -872,9 +878,9 @@ void game_update()
 			//detect the onset of hand movement, for calculating latency
 			if (!mvtStarted && (player->Distance(startCircle) > START_RADIUS*1.5))
 			{
-				timeMvtStart = SDL_GetTicks();
+				//timeMvtStart = SDL_GetTicks();
```

Comment:
More an example comment than anything, but do you want to delete the commented-out parts (eg. here)? It depends on whether you want to keep them around for explanation.